tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -639,3 +716,17 @@ def test_token_auth_decode_payload():
     assert 'nonce' in decoded_token
     assert 'sub' in decoded_token
     assert 'login.gov' in decoded_token.get('iss', '')
+
+
+def test_missing_django_superuser():
```

Comment:
These tests were added to cover the new lines added to settings and ensure that the appropriate errors will be raised when these environment variables are absent.